docs/SelectMulti.stories.ts

```diff
@@ -11,11 +11,11 @@ export default {
 const Template = (args, { argTypes }) => ({
 	props: Object.keys(argTypes),
 	components: { SelectMulti },
-	data() { return { values: [] } },
+	data() { return { selected: [] } },
 	template: `
 	<div class="wrapper">
 		<SelectMulti
-			v-model:values="values"
+			v-model:values="selected"
```

Comment:
I think this is what you were saying to me on Slack, but just want to be doubly sure - it's ok that `values` and `selected` don't match here (and in SelectSingle.stories.ts), right? It doesn't have to be `v-model:selected="selected"`?
I tried running your branch locally and it looks fine but just checking!

Reply:
@cafloyd that's correct.